core/templates/pages/blog-admin-page/blog-admin-page.mainpage.html

```diff
@@ -1,18 +0,0 @@
-<!DOCTYPE html>
-<html ng-app="oppia" lang="en">
-  <head>
-    <title>Oppia Blog Admin Panel</title>
-    <oppia-blog-admin-navbar></oppia-blog-admin-navbar>
-    <!-- Require head styles file using webpack -->
-    @load('pages/header_css_libs.html')
-    @load('pages/header_js_libs.html')
```

Comment:
Ditto for this and `footer_js_libs.html`!

Reply:
Not needed, as well as the `footer_js_libs.html` below

Reply:
I think these are already loaded in `oppia-root.mainpage.html` so you won't have to import it again. ([ref](https://github.com/oppia/oppia/blob/develop/core/templates/pages/oppia-root/oppia-root.mainpage.html#L4))

Reply:
Makes sense, thanks for letting me know